<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>995</x>
      <y>10</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true"/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="background">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_modif_infos">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier informations</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>231</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_creer_album">
       <property name="text">
        <string>Créer album</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ouvrir_album">
       <property name="text">
        <string>Ouvrir album</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ViewAlbums">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ViewAlbums</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertAlbumSQL">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Album</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionAjouter_Fichier"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionEditer_les_informations"/>
    <addaction name="actionEditer_l_image"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menuAlbum">
    <property name="title">
     <string>Album</string>
    </property>
    <addaction name="actionOuvrir_un_album"/>
    <addaction name="actionCr_er_un_album"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAlbum"/>
   <addaction name="menuOption"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionRogner">
   <property name="text">
    <string>Rogner</string>
   </property>
  </action>
  <action name="actionOrdonner">
   <property name="text">
    <string>Ordonner</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action75">
   <property name="text">
    <string>75%</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action25">
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="actionS_lectionner">
   <property name="text">
    <string>Sélectionner</string>
   </property>
  </action>
  <action name="actionAjouter_Fichier">
   <property name="text">
    <string>Ajouter Fichier</string>
   </property>
  </action>
  <action name="actionOuvrir_un_album">
   <property name="text">
    <string>Ouvrir un album</string>
   </property>
  </action>
  <action name="actionCr_er_un_album">
   <property name="text">
    <string>Créer un album</string>
   </property>
  </action>
  <action name="actionEditer_les_informations">
   <property name="text">
    <string>Editer les informations</string>
   </property>
  </action>
  <action name="actionEditer_l_image">
   <property name="text">
    <string>Editer l'image</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>?</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
